lib/ipmi_fru.c

```diff
@@ -4095,7 +4095,7 @@ ipmi_fru_get_internal_use_info(  struct ipmi_intf * intf,
 		return -1;
 	}
 
-	memset(&fru, 0, sizeof(fru));
+	memset(fru, 0, sizeof(struct fru_info));
```

Comment:
In fact, this line is superfluous as exactly this is already done in line 4072 above. Please just remove it here.

Reply:
Thanks for your reply. I have deleted this superfluous line, but how I update this pull request?